docs/listings/deps_run_t6.txt

```diff
@@ -1,30 +1,41 @@
 [ReFrame Setup]
-  version:           3.10.0-dev.3+149af549
+  version:           4.0.0-dev.1+36ffa085
   command:           './bin/reframe -c unittests/resources/checks_unlisted/deps_complex.py -n T6 -r'
-  launched by:       user@host
+  launched by:       user@tresa.local
   working directory: '/home/user/Repositories/reframe'
-  settings file:     '<builtin>'
+  settings file:     '/home/user/Repositories/reframe/tutorials/config/settings.py'
```

Comment:
This should have been run without config based on gendoclistings. Maybe you had the env var before running the script?